README.md

```diff
@@ -188,7 +188,10 @@ Data for [WebAssembly](https://developer.mozilla.org/docs/WebAssembly) features.
 
 ### [`webdriver`](./webdriver)
 
-Data for [WebDriver](https://developer.mozilla.org/en-US/docs/Web/WebDriver) features.
+Data for [WebDriver](https://developer.mozilla.org/en-US/docs/Web/WebDriver) features, including:
+
+- `bidi` - WebDriver BiDi protocol
+- `commands` - WebDriver Classic protocol
```

Comment:
Note that `commands` will be renamed to `classic` in https://github.com/mdn/browser-compat-data/pull/25802.

Reply:
As soon as both PRs are approved, I will merge them together.